openapi3/management.yaml

```diff
@@ -13930,7 +13930,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/UpdateUserRequest'
+              $ref: '#/components/schemas/User'
```

Comment:
Will this require broader review, or is this somehow specific to just the .NET Sdk? Or, has it already been reviewed?